versioned_sidebars/version-8.0-sidebars.json

```diff
@@ -253,6 +253,9 @@
             {
               "Out-of-the-box Connectors": [
                 "components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview",
+                "components/integration-framework/connectors/out-of-the-box-connectors/aws-sqs",
+                "components/integration-framework/connectors/out-of-the-box-connectors/googledrive",
+                "components/integration-framework/connectors/out-of-the-box-connectors/aws-lambda",
```

Comment:
I'm not sure if this is the area, but... having the Connectors alphabetized on the side panel isn't there in "Next". Could we update this PR to include the proper order: 
1. Amazon SQS Connector
2. AWS Lambda Connector
3. Google Drive Connector
4. REST Connector
5. SendGrid Connector
6. Slack Connector

cc @MaxTru @igpetrov

Reply:
Right. Thanks. Corrected